src/components/@shared/AssetList/index.tsx

```diff
@@ -52,14 +52,15 @@ export default function AssetList({
   useEffect(() => {
     if (!assets) return
     setAssetsWithPrices(assets as AssetExtended[])
-    setLoading(false)
+    // setLoading(true)
```

Comment:
this makes all AssetList show the loader way longer to user than it should be, as by removing this assets will only be visible after the price has been fetched. Everything else to display AssetTeaser we have before fetching the prices